lib/ast/AST.cpp

```diff
@@ -132,34 +136,46 @@ ValueType KORECompositeSort::getCategory(KOREDefinition *definition) {
 }
 
 std::string KORECompositeSort::getHook(KOREDefinition *definition) {
-  auto &att = definition->getSortDeclarations().at(this->getName())->getAttributes();
+  auto &att
+      = definition->getSortDeclarations().at(this->getName())->getAttributes();
   if (!att.count("hook")) {
     return "STRING.String";
   }
   auto &hookAtt = att.at("hook");
   assert(hookAtt->getArguments().size() == 1);
-  auto strPattern = dynamic_cast<KOREStringPattern *>(hookAtt->getArguments()[0].get());
+  auto strPattern
+      = dynamic_cast<KOREStringPattern *>(hookAtt->getArguments()[0].get());
   return strPattern->getContents();
 }
 
 ValueType KORECompositeSort::getCategory(std::string name) {
   SortCategory category;
   uint64_t bits = 0;
-  if (name == "MAP.Map") category = SortCategory::Map;
-  else if (name == "LIST.List") category = SortCategory::List;
-  else if (name == "SET.Set") category = SortCategory::Set;
-  else if (name == "ARRAY.Array") category = SortCategory::Symbol; // ARRAY is implemented in K
-  else if (name == "INT.Int") category = SortCategory::Int;
-  else if (name == "FLOAT.Float") category = SortCategory::Float;
-  else if (name == "BUFFER.StringBuffer") category = SortCategory::StringBuffer;
-  else if (name == "BOOL.Bool") category = SortCategory::Bool;
-  else if (name == "KVAR.KVar") category = SortCategory::Variable;
-  // we expect the "hook" of a MInt to be of the form "MINT.MInt N" for some bitwidth N
+  if (name == "MAP.Map")
+    category = SortCategory::Map;
+  else if (name == "LIST.List")
+    category = SortCategory::List;
+  else if (name == "SET.Set")
+    category = SortCategory::Set;
+  else if (name == "ARRAY.Array")
+    category = SortCategory::Symbol; // ARRAY is implemented in K
+  else if (name == "INT.Int")
+    category = SortCategory::Int;
+  else if (name == "FLOAT.Float")
+    category = SortCategory::Float;
+  else if (name == "BUFFER.StringBuffer")
+    category = SortCategory::StringBuffer;
+  else if (name == "BOOL.Bool")
+    category = SortCategory::Bool;
+  else if (name == "KVAR.KVar")
+    category = SortCategory::Variable;
+  // we expect the "hook" of a MInt to be of the form "MINT.MInt N" for some
+  // bitwidth N
```

Comment:
This was one of the only places I thought you might strongly prefer the pre-formatted version. There's an option in the next release of clang-format that would get you the one-line version; for now I can mark this block as not for formatting, or leave it as it is.